index.html

```diff
@@ -384,6 +387,8 @@ <h2>Glossary</h2>
 
 <p><dfn class="lint-ignore export" data-lt="percent encoding|%HH encoding|percent encoded">Percent-encoding</dfn>. Percent encoding is the escaping mechanism defined by URI [[RFC3986]] for the encoding of arbitrary byte values not otherwise permitted into a URI. For example, if a user wishes to include the character <span class="codepoint" translate="no"><bdi lang="und">&#x2F;</bdi><span class="uname">U+002F SOLIDUS</span></span> in a URI, the byte <code>0x2F</code> is encoded as the character sequence <code>%2F</code>. If the user wishes to include the character <span class="codepoint" translate="no"><bdi lang="en">&#x00E9;</bdi><span class="uname">U+00E9 LATIN SMALL LETTER E WITH ACUTE</span></span> in a URI, the <a>UTF-8</a> byte sequence for this character (<code>0xC3 0xA9</code>) could be encoded as the sequence <code>%C3%A9</code>.</p>
 
+<p><dfn class="lint-ignore export">Plane</dfn>. <a href="https://unicode.org/glossary/#plane" target="_blank">Unicode definition</a>: <q cite="https://unicode.org/glossary/#plane">A range of 65,536 (10000<sub>16</sub>) contiguous Unicode code points, where the first code point is an integer multiple of 65,536 (10000<sub>16</sub>). Planes are numbered from 0 to 16, with the number being the first code point of the plane divided by 65,536. Thus Plane 0 is U+0000..U+FFFF, Plane 1 is U+<b>1</b>0000..U+<b>1</b>FFFF, ..., and Plane 16 (10<sub>16</sub>) is U+<b>10</b>0000..<b>10</b>FFFF. (Note that ISO/IEC 10646 uses hexadecimal notation for the plane numbers—for example, Plane B instead of Plane 11).</q> See also [=Basic Multilingual Plane=].</p>
```

Comment:
```suggestion
<p><dfn class="lint-ignore export">Plane</dfn>. <a href="https://unicode.org/glossary/#plane" target="_blank">Unicode definition</a>: <q cite="https://unicode.org/glossary/#plane">A range of 65,536 (2<sub>16</sub>) contiguous Unicode code points, where the first code point is an integer multiple of 65,536 (2<sub>16</sub>). Planes are numbered from 0 to 16, with the number being the first code point of the plane divided by 65,536. Thus Plane 0 is U+0000..U+FFFF, Plane 1 is U+<b>1</b>0000..U+<b>1</b>FFFF, ..., and Plane 16 (10<sub>16</sub>) is U+<b>10</b>0000..<b>10</b>FFFF. (Note that ISO/IEC 10646 uses hexadecimal notation for the plane numbers—for example, Plane B instead of Plane 11).</q> See also [=Basic Multilingual Plane=].</p>
```

Reply:
also what mean by `Plane 16 (10<sub>16</sub>)`?

Reply:
I think this means 10000 in hexadecimal notation, not 2 to power of 16 (which should be `2<sup>16</sup>`).

Reply:
ah,,, I've misread sub with sup... sorry.